README.md

```diff
@@ -177,6 +177,8 @@ Windows                  8 hrs 46 mins       ███████████
 
 `SHOW_PROJECTS` flag can be set to `False` to hide the Projects worked on
 
+`IGNORED_REPOS`  flag can be set to `"waka-readme-stats, my-first-repo"` (just an example) to ignore some repos you don’t want to be appear here
```

Comment:
Thank you for your PR, but this line is a duplicate of line 238.😀

Reply:
Wanted to document the feature without altering someone else's documentation. I have now done so